trytravis.py

```diff
@@ -320,9 +385,17 @@ def _slug_from_url(url):
 
 def _version_string():
     """ Gets the output for `trytravis --version`. """
+    """ Gets python version before get system information,
+    because methods like platorm.dist is not disponible
+    in python 3.8 and 3.9. """
+    py_version = sys.version[:5]
     platform_system = platform.system()
     if platform_system == 'Linux':
-        os_name, os_version, _ = platform.dist()
+        if '3.8' or '3.9' in py_version:
```

Comment:
This change doesn't look related to the Travis v3 API. Could you revert it?

Also: This doesn't do what you think it does. This statement always evaluates to a truthy value.
I think you meant `'3.8' in py_version or '3.9' in py_version`?

Is there any reason in particular the previous implementation doesn't work?

Reply:
This project are builds in various python version, in version 3.8 and 3.9 this library has no method dist(), and i verify python installed version and if 3.8 or 3.9 i use outher commands to get platform info. Without this build fails in 3.8 and 3.9(nightly)
https://docs.python.org/3.8/library/platform.html
https://docs.python.org/3.9/library/platform.html